docs/architecture/adr-043-interchain-identifiers.md

```diff
@@ -0,0 +1,108 @@
+
+IIDs also enable (off-chain) assertions to be made about (on-chain) digital assets – for instance, in the form of Verifiable Credentials.
+
+An IID Method DID is **self-referential**, which means the asset describes and locates itself in its chain namespace.
+
+Compare this with a generic, all-purpose DID which can be used to identify and refer to _any_ entity. For instance, the canonical application of DIDs is Self-Sovereign Identity, which provides people and other entities with identifiers together with cryptographic methods to authenticate as the subject and to exercise control over the state of the DID Document.
+
+---
+We define **Interchain Identifiers** as being explicitly purposed to identify and locate assets within a chain namespace.
+
+---
+
+- An asset on a Cosmos chain <-------- Cosmos IID identifies and locates
+- Cosmos DID --------> Identifies and locates an external thing
+
+All DIDs have an associated **DID Document**, which should subscribe to the W3C DID-Core standard.
+This defines the cryptographic control structure for a DID, and may also specify the Service types and endpoints which are related to the DID subject.
+
+The properties of a DID-Core standard DID Document include:
+- **Identifier** (the DID) of the subject with the Identifier of the Controller who is authorized to update the DID Document.
+- **Verificaton Method/s** (The cryptographic key material).
+- **Verification Relationship/s** (Describing and restricting the purposes for which the verification Methods can be used, such as Authentication).
+- **Services** related to the DID subject, with endpoints for reaching these services, such as data stores or blockchain services.
+
+For the IID Method, two additional DID Document properties have been specified, which are required for many types of on-chain assets:
+- **Linked Resources** for describing the resources which are associated with an on-chain asset, such as a media file together with its resource identifier, type, format, location and cryptographic proof (such as a hash). 
+- **Accorded Rights** for recording machine-executable capabilities, legal rights, or any other entitlements, which are accorded to the owner/controller of an asset.
+
+Linked Resources can demonstrate the distinctions between an IID and DID:
+```
+Asset IID   // self-referrant <--
+├── Linked Resources  // examples
+│   ├── Resource DID  // externally referrant -->
+│   ├── Other Resource URL
+│   ├── Another Resource CID (Content ID)
+```
+
+The **DID Syntax** for an IID conforms to the [DID-Core Syntax](), which would be expressed as follows for an asset minted from the NFT Module on a Cosmos chain:
+
+`DID:COSMOS:CHAIN:MODULE:abc123`
+
+Where: 
+- `COSMOS` is the method _(it is not the Cosmos Zone namespace)_.
+- `CHAIN` is the namespace, such as `IXO` or `REGEN` _(it is not the chain-id)_.
```

Comment:
My expectation is that this would be an on-chain registry on the Hub with some sort of governance mechanism for managing name claims.

As for who would build or control that, I'm not sure who the best party would be, but I understand there are a few options available.

Reply:
Good, let's add it in the doc.